corehq/ex-submodules/pillowtop/es_utils.py

```diff
@@ -52,6 +52,7 @@ def _get_analysis(*names):
     'default': {
         "settings": {
             "number_of_replicas": 0,
+            "number_of_shards": 5,
```

Comment:
@esoergel @millerdev FYI. On ES2 the default number of shards used to be 5 (so, all of our indexes have 5 shards unless customized via settings.ES_SETTINGS). ES7 default shards is just 1, so we will have to set this explicitly to 5 to have a similar scale up as existing clusters on all envs..